File: Src/java/poc/translator/src/main/java/org/cqframework/cql/poc/translator/model/LibraryRef.java
package org.cqframework.cql.poc.translator.model;

import org.hl7.elm.r1.Expression;

// Note: This class is only used as a place-holder during resolution in a translator (or compiler...)
public class LibraryRef extends Expression {
    private String libraryName;

    public String getLibraryName() {
        return libraryName;
    }

    public void setLibraryName(String value) {
        libraryName = value;
    }
}

File: Src/java/cql-to-elm/src/commonMain/kotlin/org/cqframework/cql/cql2elm/model/LibraryRef.kt
package org.cqframework.cql.cql2elm.model

import org.hl7.elm.r1.Expression

// Note: This class is only used as a place-holder during resolution in a translator (or
// compiler...)
class LibraryRef(localId: String, val libraryName: String?) : Expression() {
    init {
        this.localId = localId
    }
}
